scripts/core/superdesk-api.d.ts

```diff
@@ -879,6 +879,7 @@ declare module 'superdesk-api' {
         saml_auth: any;
         google_auth: any;
         saml_label: any;
+        disallowed_characters: any;
```

Comment:
Always define proper types when writing new code. There are still many places where it's set to `any`. It's because we did it for most properties when migrating and didn't update it yet. We shouldn't use `any` for new code unless there are incompatibilities when integrating something will older code.